Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -84,14 +86,15 @@ public override void Register(IGitUICommands gitUiCommands)
 
         private void UpdateJiraSettings()
         {
+            if (!enabledSettings[Settings].GetValueOrDefault())
```

Comment:
enabledSettings.ValueOrDefault(Settings) - this way you get the default value passed to the setting constructor.
The lines 100 and 101 should be updated accordingly.